### PR TITLE
Updated motr in prose for image resizing and alignment

### PR DESCRIPTION
**Pull request by swatid-seagate**
# Problem Statement
- Updating motr-in-prose.md for image resizing and alignment 

# Review Checklist 
  Checklist for Author
- [*] PR is self reviewed
- [*] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [*] Changes done to motr-in-prose.md file as per suggestion

-----
[View rendered doc/motr-in-prose.md](https://github.com/swatid-seagate/cortx-motr/blob/motr-in-prose-patch/doc/motr-in-prose.md)

### CODE REVIEW

**Comment by andriytk** [comment]
Please, fill the PR description template or - just remove unneeded clauses.

In any case - the template should not be left empty.

File: doc/motr-in-prose.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ suggestions at the same detail level.
 # overview, idea
 
 Motr is a distributed storage system, targeting 
-[exascale](https://en.wikipedia.org/wiki/Exascale_computing) configurations. 
+[exascale](https://en.wikipedia.org/wiki/Exascale_computing) configurations.
```
Comment (andriytk) [inline doc/motr-in-prose.md:59]: Why?

**Review by andriytk — APPROVED** [review]
The pics look much better now. Thanks!

I approve the change, but again: please, update the PR description.

File: doc/motr-in-prose.md (hunk 1 above)
Comment (swatid-seagate) [reply doc/motr-in-prose.md:59]: That was by mistake.

File: doc/motr-in-prose.md [hunk 2]
```diff
@@ -125,8 +125,9 @@ network messages.
 Distributed transaction manager ([dt](#dtm-idea)) allows operations to be 
 grouped into distributed transactions, which are guaranteed to be atomic in 
 the face of certain failures.
-
-<img src="/doc/Images/1-Distributed-transaction-manager.JPG" width=600 height=500>
+<p align="center">
+<img src="/doc/Images/1-Distributed-transaction-manager.JPG" width=400 height=300>
+</p>
```
Comment (nikitadanilov) [inline doc/motr-in-prose.md:130]: This does not look right. Markdown, by design, does not specify the placement of textual and graphical elements: these decisions are left to the various markdown "processors" (like html backend). Markdown documents are supposed to be readable as text, and adding html markup to them makes them unreadable.

**Review by nikitadanilov — CHANGES_REQUESTED** [review]

File: doc/motr-in-prose.md (hunk 2 above)
Comment (swatid-seagate) [reply doc/motr-in-prose.md:130]: @andriytk, tagging you to have a thought on Nikita's suggestion.

Comment (andriytk) [reply doc/motr-in-prose.md:130]: @nikitadanilov, do you have any suggestion on how to make them look better in github processor? (Which is the one most people use by default most of the time to read the document.)

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete

**Pull request closed without merging**